ci/docker/min.build.wanda.yaml

```diff
@@ -4,6 +4,9 @@ dockerfile: ci/docker/min.build.Dockerfile
 srcs:
   - ci/env/install-dependencies.sh
   - ci/env/install-core-prerelease-dependencies.sh
+  - python/setup.py
+  - python/ray/_version.py
+  - README.rst
```

Comment:
this feels quite hackery and unstable.. this does not guarantee to have the same behavior as `pip install ray`

also if the purpose of the min build is to probe Internet, then it just should not use wanda to install the deps, as not using the cache will defeat the purpose.

and it should skip on premerge (or needs opt-in) and postmerge only. we do not want changes on the Internet to block all developers.

Reply:
yes let's discuss in the other meeting